### PR TITLE
*: Add copyright header

### LINKED ISSUE #23: [enhancement] Add copyright header to source code
/kind enhancement
/priority p3

### PR DESCRIPTION
**Pull request by gaocegege**
Signed-off-by: Ce Gao <gaoce@caicloud.io>

**What this PR does / why we need it**:

Add copyright

**Which issue(s) this PR is related to** *(optional, link to 3rd issue(s))*:

Fixes #23 

**Special notes for your reviewer**:

/cc @ddysher 

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: cmd/kubeflow-kernel/command/root.go [hunk 1]
```diff
@@ -1,3 +1,17 @@
+// Copyright [yyyy] [name of copyright owner]
```
[the remark this answers is not in the document]
Comment (ddysher) [reply cmd/kubeflow-kernel/command/root.go]: SGTM

[the remark this answers is not in the document]
Comment (gaocegege) [reply cmd/kubeflow-kernel/command/root.go]: Addressed

**Comment by gaocegege** [comment]
The coveralls is 抽风的, the actual coverage of the project should be 48%. Sometimes coveralls will tell us the coverage is 97%. I am not sure whether goveralls or coveralls.io has the bug.

**Comment by ddysher** [comment]
ok...
> The coveralls is 抽风的, the actual coverage of the project should be 48%. Sometimes coveralls will tell us the coverage is 97%. I am not sure whether goveralls or coveralls.io has the bug.

/lgtm

I think the robot won't merge it since the status is red?

**Comment by ddysher** [comment]
I'll force merge since there seems to be some problems with coverall.

**Pull request merged**